test/WhitelistedCommitment.js

```diff
@@ -368,19 +399,81 @@ contract(
         );
       });
 
+      it("should send all neumarks and zero ticket when neumarks remainder below threshold", async () => {
+        const startingDate = closeFutureDate();
+        const investor1 = accounts[0];
+        const fixedInvestors = [investor1];
+        const fixedDeclaredTickets = [etherToWei(2)];
+
+        const {
+          commitment,
+          lockedAccount,
+          neumark,
+          curve
+        } = await deployAllContracts(lockAdminAccount, whitelistAdminAccount, {
+          commitmentCfg: {
+            fixedInvestors,
+            fixedTickets: fixedDeclaredTickets,
+            startTimestamp: startingDate,
+            minTicket: etherToWei(0),
+            eurEthRate: etherToWei(0.1) // sets up a rate allowing for rounding errors
+          }
+        });
+
+        const expectedNeumarkAmmount = await curveInEther(
+          fixedDeclaredTickets[0],
+          await commitment.ethEURFraction()
```

Comment:
it's the same as `eurEthRate: etherToWei(0.1)` above. Could you extract and reuse that value? I want to avoid getting data from contracts that later we would assert on to minimize the possibility of hitting a bug from contract in test.

Reply:
ok done